data/maps/Route104/map.json

```diff
@@ -22,11 +22,6 @@
       "direction": "down",
       "offset": 0,
       "map": "MAP_ROUTE105"
-    },
-    {
-      "direction": "right",
-      "offset": 50,
-      "map": "MAP_PETALBURG_CITY"
     }
```

Comment:
Part of a change that lets you walk from petalburg city into rewind route 1. You can remove this without causing any errors